src/components/AddressButton/AddressButton.tsx

```diff
@@ -40,44 +58,48 @@ const AddressButton: React.FC<AddressButtonProps> = ({ name, address, to, uniswa
   }, [darkMode, uniswap]);
 
   const DisplayAddress = useMemo(() => {
-    if (uniswap) {
-      return (
-        <>
-          <span className="address combine">
-            <AddressStart>{address}</AddressStart>
-            <AddressEnd>{address}</AddressEnd>
-          </span>
-        </>
-      );
-    } else {
-      return (
-        <>
-          <span className="address">{address}</span>
-        </>
-      );
-    }
+    return (
+      <>
+        <span className="address">{address}</span>
+      </>
+    );
   }, [darkMode, uniswap]);
 
   return (
     <>
       <Box row>
-        <StyledButton darkMode={darkMode} uniswap={uniswap}>
-          <StyledSpan>
-            <StyledName darkMode={darkMode} uniswap={uniswap}>
-              {name ? name + " " : ""}
-            </StyledName>
-            <StyledLink darkMode={darkMode} color="hsl(339deg 89% 49% / 100%)" href={"https://etherscan.io/address/" + address} target="_blank">
-              {DisplayAddress}
-            </StyledLink>
-            <StyledCopy
-              darkMode={darkMode}
-              uniswap={uniswap}
-              onClick={() => {
-                navigator.clipboard.writeText(address ? address : "");
-              }}
-            ></StyledCopy>
-          </StyledSpan>
+        <StyledButton
+          darkMode={darkMode}
+          uniswap={uniswap}
+          href={"https://etherscan.io/address/" + address}
+          target="_blank"
+        >
+          <StyledName darkMode={darkMode} uniswap={uniswap}>
+            {name ? name + " " : ""}
+          </StyledName>
+          <StyledAddress darkMode={darkMode} color="hsl(339deg 89% 49% / 100%)">
+            {DisplayAddress}
+          </StyledAddress>
         </StyledButton>
+        <StyledCopy
+          darkMode={darkMode}
+          uniswap={uniswap}
+          data-tip
+          data-for="copyTip"
+          onClick={() => {
+            navigator.clipboard.writeText(address ? address : "");
+          }}
+        ></StyledCopy>
+        <ReactTooltip
+          id="copyTip"
+          type="dark"
+          arrowColor="transparent"
+          place="top"
+          effect="solid"
+          backgroundColor="grey"
+        >
+          Copy Address
+        </ReactTooltip>
```

Comment:
nice tooltip addition but doesnt look good, you can use [what we have](https://github.com/yam-finance/yam-www/blob/master/src/components/FancyValue/FancyValue.tsx#L131-L151) on the dashboard, changing the color to something more theme oriented